perun-core/src/main/java/cz/metacentrum/perun/core/entry/UsersManagerEntry.java

```diff
@@ -972,6 +975,73 @@ public boolean isLoginAvailable(PerunSession sess, String loginNamespace, String
 		return getUsersManagerBl().isLoginAvailable(sess, loginNamespace, login);
 	}
 
+	@Override
+	public List<Pair<String, String>> getAllBlockedLoginsInNamespaces(PerunSession sess) throws PrivilegeException {
+		Utils.checkPerunSession(sess);
+
+		if (!AuthzResolver.authorizedInternal(sess, "getAllBlockedLoginsInNamespaces_policy")) {
+			throw new PrivilegeException(sess, "getAllBlockedLoginsInNamespaces");
+		}
+
+		return getUsersManagerBl().getAllBlockedLoginsInNamespaces(sess);
+	}
+
+	@Override
+	public boolean isLoginBlocked(PerunSession sess, String login) throws PrivilegeException {
+		Utils.checkPerunSession(sess);
+
+		if (!AuthzResolver.authorizedInternal(sess, "isLoginBlocked_policy")) {
```

Comment:
Change to match added parameter types in policy names.
```suggestion
		if (!AuthzResolver.authorizedInternal(sess, "isLoginBlocked_String_policy")) {
```